### PR TITLE
Upload invoice (49022)

### PR DESCRIPTION
**Pull request by bogdan202**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. <br> Every detail helps: versions, browser/server configuration, specific module/theme, etc. Feel free to add more information below this table.
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes 49022
| How to test?  | Please indicate how to best verify that this PR is correct.

### CODE REVIEW

File: src/OrderInvoiceSync/Hub.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<?php
```
Comment (clotairer) [inline src/OrderInvoiceSync/Hub.php:1]: Header licence

File: views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl [hunk 2]
```diff
@@ -134,6 +134,48 @@
...
+                        <th>
+                            {l s='Send invoices' mod='shoppingfeed'}
+                        </th>
+                    </tr>
+                    </thead>
+
+                    <tbody>
+                    {foreach from=$input.marketplaces item=marketplace}
+                        <tr>
+                            <td>{$marketplace->getName()}</td>
```
Comment (clotairer) [inline views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl:161]: escape

File: views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro_178.tpl [hunk 3]
```diff
@@ -134,6 +134,48 @@
...
+                            <th>
+                                {l s='Send invoices' mod='shoppingfeed'}
+                            </th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        {foreach from=$input.marketplaces item=marketplace}
+                            <tr>
+                                <td>{$marketplace->getName()}</td>
```
Comment (clotairer) [inline views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro_178.tpl:161]: escape

**Review by clotairer — CHANGES_REQUESTED** [review]

File: src/OrderInvoiceSync/Hub.php (hunk 1 above)
Comment (bogdan202) [reply src/OrderInvoiceSync/Hub.php:1]: fixed

File: views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl (hunk 2 above)
Comment (bogdan202) [reply views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl:161]: fixed

File: views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro_178.tpl (hunk 3 above)
Comment (bogdan202) [reply views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro_178.tpl]: fixed

File: views/tmp/index.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (clotairer) [inline views/tmp/index.php:1]: file to remove. generated by autoindex

File: views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl (hunk 2 above)
Comment (clotairer) [reply views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl:161]: OK in order_status_syncro_178.tpl but not in order_status_syncro.tpl

**Review by clotairer — CHANGES_REQUESTED** [review]

File: views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl (hunk 2 above)
Comment (bogdan202) [reply views/templates/admin/shoppingfeed_order_import_rules/order_status_syncro.tpl]: fixed

File: views/tmp/index.php (hunk 4 above)
Comment (bogdan202) [reply views/tmp/index.php]: removed

**Review by clotairer — APPROVED** [review]

**Pull request merged**